compiler/ilgen/TypeDictionary.hpp

```diff
@@ -415,46 +416,55 @@ class TypeDictionary
-   TR::IlType       * VectorFloat;
-   TR::IlType       * VectorDouble;
-
-   TR::IlType       * _pointerToPrimitiveType[TR::NumOMRTypes];
-   TR::IlType       * pNoType;
-   TR::IlType       * pInt8;
-   TR::IlType       * pInt16;
-   TR::IlType       * pInt32;
-   TR::IlType       * pInt64;
-   TR::IlType       * pWord;
-   TR::IlType       * pFloat;
-   TR::IlType       * pDouble;
-   TR::IlType       * pAddress;
-   TR::IlType       * pVectorInt8;
-   TR::IlType       * pVectorInt16;
-   TR::IlType       * pVectorInt32;
-   TR::IlType       * pVectorInt64;
-   TR::IlType       * pVectorFloat;
-   TR::IlType       * pVectorDouble;
+   TR::IlType                                             * _primitiveType[TR::NumOMRTypes];
+   TR::IlType                                             * NoType;
+   TR::IlType                                             * Int8;
+   TR::IlType                                             * Int16;
+   TR::IlType                                             * Int32;
+   TR::IlType                                             * Int64;
+   TR::IlType                                             * Word;
+   TR::IlType                                             * Float;
+   TR::IlType                                             * Double;
+   TR::IlType                                             * Address;
+   TR::IlType                                             * VectorInt8;
+   TR::IlType                                             * VectorInt16;
+   TR::IlType                                             * VectorInt32;
+   TR::IlType                                             * VectorInt64;
+   TR::IlType                                             * VectorFloat;
+   TR::IlType                                             * VectorDouble;
+
+   TR::IlType                                             * _pointerToPrimitiveType[TR::NumOMRTypes];
+   TR::IlType                                             * pNoType;
+   TR::IlType                                             * pInt8;
+   TR::IlType                                             * pInt16;
+   TR::IlType                                             * pInt32;
+   TR::IlType                                             * pInt64;
+   TR::IlType                                             * pWord;
+   TR::IlType                                             * pFloat;
+   TR::IlType                                             * pDouble;
+   TR::IlType                                             * pAddress;
+   TR::IlType                                             * pVectorInt8;
+   TR::IlType                                             * pVectorInt16;
+   TR::IlType                                             * pVectorInt32;
+   TR::IlType                                             * pVectorInt64;
+   TR::IlType                                             * pVectorFloat;
+   TR::IlType                                             * pVectorDouble;
```

Comment:
The abundance of formatting spaces introduced to line all these fields up with `_structsByName` and `_unionsByName` is needlessly excessive to satisfy those two fields.  I say go back to the tighter formatting that was there originally.